person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Similarly, let's mark `rental` with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.